guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>This should be a relative link to the api-reference: surround the word <em>update() </em>in square brackets followed directly by parantheses to </p><p> </p><p>You can update certain initiative properties using the [update()] (<em>/python/api-reference/arcgis.apps.hub.html#arcgis.apps.hub.initiatives.Initiative.update) method on the [initiative] (</em>/python/api-reference/arcgis.apps.hub.html#initiative<em>) object.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>